lib/build.js

```diff
@@ -8,6 +8,8 @@ const apiUri = Symbol('apiUri');
 const tokenGen = Symbol('tokenGen');
 const uiUri = Symbol('uiUri');
 const ABORT_CODE = 130; // 128 + SIGINT 2 (^C)
+const DEFAULT_BUILD_TIMEOUT = 90 * 60; // seconds
```

Comment:
Other default build timeout set as minutes [like this](https://github.com/screwdriver-cd/executor-k8s-vm/blob/b4a556659cc0ddc9d754bd060e0bfbf5b80ab5bd/index.js#L14) so I think "minutes" is better in this case.

Reply:
Thanks, I fixed!
Converts to second [here](https://github.com/screwdriver-cd/models/pull/245/files#diff-26b681bd4dac10f25d5293c7595c435bR168) instead.